src/Models/NonhydrostaticModels/nonhydrostatic_tendency_kernel_functions.jl

```diff
@@ -119,8 +123,11 @@ pressure anomaly.
 
     model_fields = merge(velocities, tracers, auxiliary_fields)
 
-    return ( - div_𝐯v(i, j, k, grid, advection, velocities, velocities.v)
-             - div_𝐯v(i, j, k, grid, advection, background_fields.velocities, velocities.v)
+    total_velocities = (u = SumOfArrays{2}(velocities.u, background_fields.velocities.u),
```

Comment:
wouldn't this give you a problem when `background_fields = NamedTuple()` (as in the default constructor)?

Reply:
Thats how it is currently done, I think the defaults get expanded out to `(velocities = (u = ZeroField(grid), ...`